docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -166,12 +166,13 @@ Note: If you are contributing a Python file without imports or function definiti
 ### Develop on `airbyte-webapp`
 
 - Spin up Airbyte locally so the UI can make requests against the local API.
-- Stop the `webapp`.
 
 ```bash
-docker-compose stop webapp
+BASIC_AUTH_USERNAME="" BASIC_AUTH_PASSWORD="" docker-compose up
 ```
```

Comment:
It's not actually necessary to stop the webapp container. IMO nicer to keep it short and omit this 🙂